logging/google/cloud/logging/logger.py

```diff
@@ -544,6 +571,8 @@ def commit(self, client=None):
                 info['httpRequest'] = http_req
             if timestamp is not None:
                 info['timestamp'] = _datetime_to_rfc3339(timestamp)
+            if trace is not None:
+                info['trace'] = trace
```

Comment:
We need coverage for this case.

Reply:
added
```
test_log_text_w_trace()
test_log_struct_w_trace()
test_log_proto_w_trace()
```

Reply:
We still don't have a testcase for `Batch.commit` with `trace` in an entry, which causes a coverage error for these two lines.